### PR TITLE
chore(pep8): Make the Backend Python PEP8 compliant and Unit Test Code Coverage

### PR DESCRIPTION
**Pull request by mannyrivera2010**
Made the code PEP8 complaint and added Unit Test Code Coverage

### CODE REVIEW

**Comment by mannyrivera2010** [comment]
Pull Request for Issue #83

**Comment by mannyrivera2010** [comment]
Please review issue and pull request.
@arw180 (Optional) 
@clarkjefcoat 
@tonyalletag

**Comment by clarkjefcoat** [comment]
So two blank lines before every class or function definition, a carriage return as the last character of every file and consistent indenting?

Looks good to me :+1:

**Comment by mannyrivera2010** [comment]
I ran this command "pep8 --statistics ozpcenter/ ozp/ ozpiwc/ --ignore=E501,E128"  and tried to fix the pep8 violations

File: ozp/settings.py [hunk 1]
```diff
@@ -27,6 +27,15 @@
 
 ALLOWED_HOSTS = []
 
+# Use nose to run all tests
+TEST_RUNNER = 'django_nose.NoseTestSuiteRunner'
+
+# Tell nose to measure coverage on the 'foo' and 'bar' apps
```
Comment (arw180) [inline ozp/settings.py]: fix comment

**Comment by arw180** [comment]
Glad to see the pep8 violations fixed. You're right to add the two database migration files, though ideally that would have been in a separate PR. Same with the `run_test.sh` script, though I wouldn't worry about it this time around. 

Also, you might want to consider adding the pep8 check to the Jenkins job so that the build fails if a pep8 violation is checked in. 

Otherwise, looks good :+1:

File: ozp/settings.py (hunk 1 above)
Comment (mannyrivera2010) [inline ozp/settings.py]: Fixed in d98d8c2

**Comment by mannyrivera2010** [comment]
I agree with you that they should have been different PRs.  The scope this PR did not include that. I will separate better.   

I added this ticket to the issues backlog for jenkins.
````
Add the pep8 check to the Jenkins job so that the build fails if a pep8 violation is checked in. #85
````

**Pull request merged**